src/config.rs

```diff
@@ -2526,6 +2538,14 @@ impl TiKvConfig {
         self.backup.validate()?;
         self.pessimistic_txn.validate()?;
         self.gc.validate()?;
+
+        if self.storage.enable_flow_control {
+            self.raftdb.defaultcf.disable_write_stall = true;
```

Comment:
Why disable raftdb?

Reply:
In the case where raftdb stalls, kvdb should be flow controlled first, cause kvdb usually has more compaction flow comparing to raftdb.

Reply:
I think we should keep it enabled for now. I came across some practical cases where raftdb has much more compaction than kvdb (can't recall the exact benchmark setting though), other than that, in some extreme cases (raftdb disk slow, cpu throttle) raftdb stall can be useful to avoid OOM.

Reply:
OOM or disk full caused by raftdb's accumulation should be considered by another mechanism now.

Reply:
I don't think the OOM flow control will consider RocksDB memory..

Reply:
Anyway I just don't see the downside of using raftdb write stall as a safety net.

Reply:
But I can't see any circumstance that RaftDB stalls but KvDB doesn't. Consider the case that both KvDB and RaftDB has a lot of pending compaction bytes(no matter which has more), flow control begins to increase discardable ratio to control the accumulation, but it won't surpass it under threshold. Whereas, RaftDB triggers write stall now and affects raftstore.

Reply:
If you are so worried about pending compaction bytes, you can simply set its limit to 0. I'm more concerned about memtable.